adapters/sidekiq/lib/opentelemetry/adapters/sidekiq/middlewares/client/tracer_middleware.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Adapters
+    module Sidekiq
+      module Middlewares
+        module Client
+          class TracerMiddleware
+            def call(worker_class, job, _queue, _redis_pool)
```

Comment:
We should include the queue name as an attribute. https://github.com/open-telemetry/opentelemetry-specification/pull/418 calls this `messaging.destination`.